source/content/posts/CoreData.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "CoreData"
+date: 2018-01-10T15:06:30Z
+draft: false
+---
+
+# Usando CoreData por primera vez
+
+Hasta hace algunos días no había tenido la necesidad de utilizar un manejador de base de datos en las aplicaciones que he desarrollado, para manejar la persistencia de datos utilizaba los [UserDefaults](https://developer.apple.com/documentation/foundation/userdefaults).
+<p align="justify">
+Pero me ví con la necesidad de almacener una cantidad de datos mucho mayor, podría utilizar los **UserDefaults** pero no es óptimo, los mismos desarrolladores de **Apple** recomiendan implementar los UserDefaults para almacenar información puntual acerca del usuario y sus preferencias, por lo que necesitaba utilizar un manejador de base de datos y esto era algo nuevo para mí, tenía que decidir entre las diferentes bases de datos compatibles con dispositivos iOS y una entre ellas se encontraba **CoreData** que es un framework desarrollada y recomendada por **Apple**.</p>
+<p align="justify">
+Dicho lo anterior, vamos a comenzar a crear nuestro modelo de datos con sus respectivas entidades, almacenaremos datos en ellos y recuperaremos esos datos para mostrarlos al usuario haciendo uso de **CoreData**</p>
+
+
+### Creando nuestro modelo de datos
+
+Crear el modelo para nuestra base de datos es tan sencillo como hacer un **click** en la opción "Use Core Data" al momento de crear tu proyecto.
+
+![Default Image](../CoreData/CrearModeloCoreData.png)
+<p align="justify">
+ Puedes ubicar el modelo de tu base de datos en bundle de tu proyecto , el nombre al modelo asigna automáticamente como **"Nombre_Proyecto.xcdatamodeld"** pero si gustas puedes cambiarle el nombre.</p>
```

Comment:
I have some suggestions for you:
* proyecto. 
* el nombre al modelo es asignado automáticamente como.